webapp-django/crashstats/topcrashers/jinja2/topcrashers/topcrashers.html

```diff
@@ -16,8 +16,8 @@
                 <a href="{{ url(
                     'crashstats:topcrasher',
                     product=query['product'],
-                    versions=query['versions'],
-                    date_range_type='report',
+                    versions=';'.join(query['versions']),
```

Comment:
I'm going to think out loud here for a bit.

I think there are two questions here:

1. What's the requirement?
2. Does the application handle junk data well?

Towards the requirement question, it sounds like versions will not contain non-ascii data. So we can treat them like ascii in the code.

Towards the junk data question, it depends on a bunch of things like what's the ramifications of junk data? Who's using it? How hard is it to put in junk data? How hard is it to deal with? In this particular case, we do string joining in a bunch of places so to handle junk data better, we could write a function that does that work for us and handles possible junk data better (probably just drop non-ascii versions) and then use that function everywhere. It sounds like the audience is a "more technical" audience. It sounds like the ramification is that the search doesn't work which is fine. Contrast that with the case where junk data causes junk data to be in the db which is usually not fine and also not the case here.

We could wait until it's a problem and then write up a bug and address it then. I think that's what I'd do.